x/name/client/cli/gov.go

```diff
@@ -106,7 +110,93 @@ $ %s tx gov submit-proposal param-change tx gov submit-proposal \
 		},
 	}
 
-	cmd.Flags().String(FlagOwner, "", "The owner of the new name, optional (defaults to from address")
+	cmd.Flags().String(FlagOwner, "", "The owner of the new name, optional (defaults to from address)")
+	cmd.Flags().BoolP(flagRestricted, "r", true, "Restrict creation of child names to owner only, optional (default false)")
+	// proposal flags
+	cmd.Flags().String(FlagTitle, "", "Title of proposal")
+	cmd.Flags().String(FlagDescription, "", "Description of proposal")
+	cmd.Flags().String(FlagDeposit, "", "Deposit of proposal")
+	return cmd
+}
+
+func GetModifyNameProposalCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "modify-name-proposal [name] (--owner [address]) (--restrict) [flags]",
+		Short: "Submit a modify name creation governance proposal",
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`Submit a modify name governance proposal along with an initial deposit.
+The proposal title and description must be provided through their respective flags.
+
+IMPORTANT: The creation of sub-names will be enabled by default unless the restrict flag is included.
+When included the owner must approve all child name creation unless an alterate owner is provided.
```

Comment:
The `--restricted` flag is added to the command with a default of `true` (see other comment), so this blurb is inaccurate.